assets/js/search.js

```diff
@@ -229,8 +229,8 @@ function highlightSearchTerms(search, context, wrapper = 'mark', cssClass = '')
 
 window.addEventListener('load', function() {
   const pageLanguage = elem('body').dataset.lang;
-  const searchIndexLangSlug = pageLanguage === defaultSiteLanguage ? '': `${pageLanguage}/`;
```

Comment:
@GaryFlynn, the trailing forward slash was optional because

1. The [baseURL](https://github.com/chipzoller/hugo-clarity/blob/aaf6da813c6352873d8ea4a9a5ab3de8c646f138/exampleSite/config/_default/config.toml#L3) already contained a trailing slash. 
2. The root url of the defaultLanguage is the baseURL. We added the forward slash for other languages.